inst/scripts/make-grieneisen-ts-metadata.R

```diff
@@ -0,0 +1,70 @@
+
+library(dplyr)
+library(R.utils)
+library(stringr)
+
+BiocVersion <- "3.14"
+path <- paste0("MicrobiomeDataSets/",BiocVersion,"/")
+
+df_Base <- data.frame(
+  BiocVersion = BiocVersion,
+  SourceVersion = NA,
+  Coordinate_1_based = TRUE,
+  Domain = "Bacteria",
+  TaxonomyId = "2",

```

Comment:
This refers to the host organism. Can you search for the scientific name and taxonomyID for the species in the Science paper (baboons)? Compare to the other metadata scripts, they all have host organism names here.

Reply:
I think it is `100937` for `Papio papio`.

See `GenomeInfoDb::loadTaxonomyDb()`

Reply:
Also perhaps the Science paper itself also has this info? To confirm.

Reply:
I found it as Amboseli baboon, also known as the yellow baboon or Papio cynocephalus.